### PR TITLE
Feature/frontend/website content

### PR DESCRIPTION
**Pull request by vasynizami**
Styled it as close as I could to the XD file specifications; for some reason the fonts in the XD file notes don't look the way the fonts look actually; the social media icons weren't provided

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/mycovidconnect/71)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -48,40 +74,40 @@ footer {
...
   height: auto;
   margin: 0 auto;
-  padding: 30px 0px;
-  background-color: #2f97f2;
+  padding: 24px 0px;
 }
 
 footer .footerin {
-  width: 98%;
+  /* width: 98%;
```
Comment (codecakes) [inline src/App.css:81]: use double star for opening comment `/** like this */` ; this is for all css/scss/less files here.
if you are sprinkling comments add a `TODO(<github handle): <reason for the comment`;
Once necessary step has been taken, remove that comment

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -9,52 +13,32 @@ const Footer = () => {
         <div className="footerin">
           <div className="w40">
             <h3>Join Our Community</h3>
-            <a href="https://www.google.com" title="Join Now">
+            <a href="https://forum.mycovidconnect.com/" title="Join Now">
```
Comment (codecakes) [inline src/components/Footer.jsx:16]: open in new tab `add target="_blank"`

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -9,52 +13,32 @@ const Footer = () => {
...
-                  Terms &amp; Conditions
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Press Enquiries">
-                  Press Enquiries
-                </Link>
-              </li>
-            </ul>
+            <a href="https://forum.mycovidconnect.com/d/2-about-xcov19-rapid-action-patient-dispatch-solution">
```
Comment (codecakes) [inline src/components/Footer.jsx:21]: all footer links open in new tab. make those changes everywhere

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -9,52 +13,32 @@ const Footer = () => {
...
+              About XCOV19
+            </a>
+            <a href="#">Contact Us</a>
+            <div className="social-media">
```
Comment (codecakes) [inline src/components/Footer.jsx:25]: ask kemo for media kits, icon-logos for helpful, wprn, mattermore, jogl; opensource against covid et al, ive pasted the links to these on main whatsapp group; for the icons that kemo can provide, remove these icons and add those w/ necessary name and links

File: src/components/Footer.jsx [hunk 5]
```diff
@@ -9,52 +13,32 @@ const Footer = () => {
...
-              <li>
-                <Link to="/" title="Press Enquiries">
-                  Press Enquiries
-                </Link>
-              </li>
-            </ul>
+            <a href="https://forum.mycovidconnect.com/d/2-about-xcov19-rapid-action-patient-dispatch-solution">
+              About XCOV19
+            </a>
+            <a href="#">Contact Us</a>
```
Comment (codecakes) [inline src/components/Footer.jsx:24]: change the name to Media Presence

File: src/index.css [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+/* @font-face {
+  font-family: 'Montserrat';
+  src: local('Montserrat'), url(./fonts/montserrat/Montserrat-Regular.otf) format('truetype');
+} */
+
+@font-face {
+  font-family: 'Montserrat Semi-Bold';
+  src: local('Montserrat Semi-Bold'), url(./fonts/montserrat/Montserrat-SemiBold.otf) format('truetype');
+}
```
Comment (codecakes) [inline src/index.css:24]: End of File new blank line

**Review by codecakes — CHANGES_REQUESTED** [review]
make sure 
- you've pulled latest changes from master and then push a commit.
- If you are working for a user story ticket number, fashion your future commits format like here: https://support.atlassian.com/bitbucket-cloud/docs/use-smart-commits/

**Review by codecakes — CHANGES_REQUESTED** [review]
move as many src/fonts files out and use cdn instead; for instance, instead of src/fonts/lato/Lato-Bold.ttf
try to lookup if cdn sites can serve these media. you will find them here:
- https://cdnjs.com/libraries?q=montserrat (u can search more there)
- https://www.bootstrapcdn.com/fontawesome/
- https://www.cdnfonts.com/montserrat.font (search more fonts there as per requirements)
- choose individual font packs here: https://pagecdn.com/lib/easyfonts

This will incredibly reduce the site load time even before caching;

**Review by codecakes — CHANGES_REQUESTED** [review]
In chrome, if you toggle to responsive device toolbar, you'll see for the following modes, the rendering is out of place:
- 400 x 655 responsive portrait
- 360 x 640 (galaxy s5)
- 375 x 667 iPhone 6/7/8
- iphoneX 375 x 812, both portrait and landscape
- ipad and ipad pro, portrait

maybe you can ask neha/mallory for relevant screen mockups for the ones you need help on? rest feel free to fit as per your viewing;

File: src/components/Footer.jsx [hunk 7]
```diff
@@ -9,52 +9,38 @@ const Footer = () => {
...
-                <Link to="/" title="Press Enquiries">
-                  Press Enquiries
-                </Link>
-              </li>
-            </ul>
+            <div className="links">
+              <a
+                target="_blank"
+                href="https://forum.mycovidconnect.com/d/2-about-xcov19-rapid-action-patient-dispatch-solution"
+              >
```
Comment (codecakes) [inline src/components/Footer.jsx:25]: add `target="_blank" rel="noopener"` in all links we want the [user to open externally](https://mathiasbynens.github.io/rel-noopener/) wherever there is `target="_blank"`

Also Add a todo over here somewhere like this:
 `TODO(@vasynizami) create a common js script for external links asking for conset`
Idea is like:
```javascript
 () => {
  // suspends initial behavior and asks user consent Yes/No whether they want to navigate to external llink or not? if not suspends default behaviour else continues. 
}
```

File: src/components/Footer.jsx [hunk 8]
```diff
@@ -9,52 +9,38 @@ const Footer = () => {
...
+                href="mailto:reachout@mycovidconnect.com"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Contact Us
+              </a>
+              <a href="#">Privacy Policy</a>
+            </div>
+            <div className="media-links">
+              <img src={Helpful} alt="helpful-logo" />
```
Comment (codecakes) [inline src/components/Footer.jsx:38]: @vasynizami did kemo not give out the links? Ask for relevant links as well and update

File: src/components/Nav.jsx [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+      <a
+        target="_blank"
+        href="https://forum.mycovidconnect.com/d/2-about-xcov19-rapid-action-patient-dispatch-solution"
+      >
+        About
+      </a>
+      <a href="#">Contact Us</a>
+      <button id="login">Log In</button>
+      <button id="signup">Sign Up</button>
+      {/* <script
```
Comment (codecakes) [inline src/components/Nav.jsx:15]: why is this commented out? 
The modal does not pop up on clilck.

**Review by codecakes — CHANGES_REQUESTED** [review]
Looks good but the modal does not pop on click. left comment. 
You can see the site behavior here: https://5f749a3e6ca6d40007cd2da6--dreamy-keller-b8ad36.netlify.app/#

**Review by codecakes — CHANGES_REQUESTED** [review]
do a `git merge` from master branch first; some changes need to be merged back into your branch;

**Review by andyderek — COMMENTED** [review]
Where are we at on this one?  I am available to help if needed.

**Comment by codecakes** [comment]
> Where are we at on this one? I am available to help if needed.

@andyderek pls feel free to take this up;

**Comment by codecakes** [comment]
@vasynizami push new commits here. see: https://github.com/Xcov19/mycovidconnect/pull/102#pullrequestreview-501626725

**Comment by codecakes** [comment]
@andyderek @vasynizami take a fresh merge from master as well

**Pull request closed without merging**